src/go/rpk/pkg/cli/cmd/debug/bundle.go

```diff
@@ -599,23 +600,53 @@ func saveResourceUsageData(ps *stepParams, conf *config.Config) step {
 // Queries 'pool.ntp.org' and writes a file with the reported RTT, time & precision.
 func saveNTPDrift(ps *stepParams) step {
 	return func() error {
-		const host = "pool.ntp.org"
+		const (
+			host    = "pool.ntp.org"
+			retries = 3
+		)
+
+		var (
+			response  *ntp.Response
+			localTime time.Time
+			err       error
+		)
+
+		queryNTP := func() error {
+			localTime = time.Now()
+			response, err = ntp.Query(host)
+			return err
+		}
+
+		err = retry.Do(
```

Comment:
It may be faster if we use `ntp.QueryWithOptions(host, ntp.QueryOptions{Timeout: time.Second})` and query a few hosts simultaneously (or the same host a few times simultaneously) and use the first successful response.

This retry loop is simple but does mean a 3s delay if there's some problems. Also I'd maybe use a for loop directly.

But this does solve the immediate issue, which is good, and any refinement can come later if needed.

Reply:
@twmb good point. However, the whole program's minimum execution time is bound by `top -b -n 10 -H -d 1`, which does 10 iterations with a 1-second delay between each iteration.